### PR TITLE
Add gomplate to `arkade get`

### PR DESCRIPTION
**Pull request by AmrSaber**
## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md)) - https://github.com/alexellis/arkade/issues/706

## How Has This Been Tested?

Tested using `go test ./...` command and all the tests (including the new one) are passing

## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [x] I have tested this on arm, or have added code to prevent deployment - there are binaries for arm and they are supported

### CODE REVIEW

File: pkg/get/tools.go [hunk 1]
```diff
@@ -2272,5 +2272,44 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
+
+			{{- if eq .Arch "aarch64" -}}
+			{{ $arch = "arm64" }}
+			{{- else if eq .Arch "arm64" -}}
+			{{ $arch = "arm64" }}
+			{{- else if eq .Arch "armv7l" -}}
+			{{ $arch = "armv7" }}
+			{{- else if eq .Arch "armv6l" -}}
+			{{ $arch = "armv6" }}
+			{{- end -}}
```
Comment (Shikachuu) [inline pkg/get/tools.go:2294]: I don't think armv6l should be included since it is dropped for a lot of project nowdays and we started to exclude it too.

**Review by Shikachuu — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Look at my comment above, can you fix that and do a rebase please?

**Comment by AmrSaber** [comment]
> Thanks for the contribution! Look at my comment below, can you fix that and do a rebase please?

Sure, will apply the required change. Just a quick question, you required a rebase not a merge?

**Comment by Shikachuu** [comment]
> > Thanks for the contribution! Look at my comment below, can you fix that and do a rebase please?
> 
> Sure, will apply the required change. Just a quick question, you required a rebase not a merge?

Yes we mostly use rebase in a case like yours, we like to keep the one commit/PR rule, unless you have a really good reason for it.

**Comment by AmrSaber** [comment]
@Shikachuu OK, I have applied the requested changes and did a rebase

**Review by Shikachuu — APPROVED** [review]
LGTM thanks @AmrSaber

(cc: @alexellis )

File: pkg/get/tools.go (hunk 1 above)
Comment (alexellis) [reply pkg/get/tools.go]: Agreed. Please hit "Resolve conversation" if this has been done.

**Comment by alexellis** [comment]
If you're happy @Shikachuu then I'm happy. We just need a rebase due to merging another tool just now.

**Comment by AmrSaber** [comment]
> We just need a rebase due to merging another tool just now.

Done

**Comment by alexellis** [comment]
CI approved to run, merging would happen after that.

**Pull request merged**